openpype/hosts/photoshop/plugins/publish/extract_review.py

```diff
@@ -20,54 +21,43 @@ class ExtractReview(openpype.api.Extractor):
     # Extract Options
     jpg_options = None
     mov_options = None
+    make_image_sequence = None
 
     def process(self, instance):
-        staging_dir = self.staging_dir(instance)
-        self.log.info("Outputting image to {}".format(staging_dir))
-
-        stub = photoshop.stub()
-
-        layers = []
-        for image_instance in instance.context:
-            if image_instance.data["family"] != "image":
-                continue
-            layers.append(image_instance[0])
-
-        # Perform extraction
-        output_image = "{}.jpg".format(
-            os.path.splitext(stub.get_active_document_name())[0]
-        )
-        output_image_path = os.path.join(staging_dir, output_image)
-        with photoshop.maintained_visibility():
-            if layers:
-                # Hide all other layers.
-                extract_ids = set([ll.id for ll in stub.
-                                   get_layers_in_layers(layers)])
-                self.log.debug("extract_ids {}".format(extract_ids))
-                for layer in stub.get_layers():
-                    # limit unnecessary calls to client
-                    if layer.visible and layer.id not in extract_ids:
-                        stub.set_visible(layer.id, False)
-
-            stub.saveAs(output_image_path, 'jpg', True)
+        self.staging_dir = self.staging_dir(instance)
```

Comment:
It is better to not store process data to object of publish plugins. It can easily cause few headaches when the code will need to be changed a little bit in future and also avoids issues with garbage collection.